content/reliable_tx.md

```diff
@@ -121,31 +125,31 @@ For each persisted transaction without validated result:
 
 In order to implement the transaction submission and verification best practices, applications need to perform the following actions.
 
-* Determine the signing account's next sequence number
-  * Each transaction has an account-specific sequence number.  This guarantees the order in which transactions signed by an account are executed and makes it safe to resubmit a transaction without danger of the transaction being applied to the ledger more than once.
-* Decide on a `LastLedgerSequence`
-   * A transaction's `LastLedgerSequence` is calculated from the last validated ledger sequence number.
-* Construct and sign the transaction
-  * The details of a signed transaction are persisted before submission.
-* Submit the transaction
-  * Initial results are provisional and subject to change.
-* Determine the final result of a transaction
-  * Final results are an immutable part of the ledger history.
+1. Determine the signing account's next sequence number
+    * Each transaction has an account-specific sequence number.  This guarantees the order in which transactions signed by an account are executed and makes it safe to resubmit a transaction without danger of the transaction being applied to the ledger more than once.
+3. Decide on a `LastLedgerSequence`
+     * A transaction's `LastLedgerSequence` is calculated from the last validated ledger sequence number.
+3. Construct and sign the transaction
+    * Persist the details of a signed transaction before submission.
+4. Submit the transaction
+    * Initial results are provisional and subject to change.
+5. Determine the final result of a transaction
+    * Final results are an immutable part of the ledger history.
 
-An application's means of performing these actions depends on the ripple API the application uses.  These interfaces may be any of:
+An application's means of performing these actions depends on the API the application uses.  These interfaces may be any of:
```

Comment:
*Nitpick*
Maybe something like "An application may use any of the following interfaces:"